src/district0x/ui/events.cljs

```diff
@@ -93,8 +94,8 @@
   ;; I've found the persisted connection to be extremely unreliable
   (.removeItem (.-localStorage js/window) "walletconnect")
   ;; https://lwhorton.github.io/2018/10/20/clojurescript-interop-with-javascript.html
-  (let [web3-modal-class (.. js/window -Web3Modal -default)
-        wallet-connect-provider (.. js/window -WalletConnectProvider -default)
+  (let [web3-modal-class (goog.object/get (goog.object/get js/window "Web3Modal") "default")
```

Comment:
This is simpler than setup around & creating or autogenerating extern files.

https://clojurescript.org/reference/dependencies#external-javascript-libraries
```
For simple cases where you only reference a small number of JavaScript symbols, you can also change your source code to reference code by string name. Google Closure Compiler will never rename strings, so this style will work without needing to create an externs file. 
```